### ISSUE TITLE
keygen generate apikey invalid 

### ISSUE THREAD

**Issue by Yooke** [tinode/chat#171]
### Subject of the issue
Then `keygen` generate key used HMAC salt `base64.URLEncoding`, but parse config `tinode.conf` used `base64.StdEncoding`.

### Is this a bug report of a feature request?
- [x] Bug report
- [ ] Feature request

#### Server-side
- [ ] api.tinode.co
- [ ] Your own setup:
  * platform Mac
  * version of tinode server, e.g. `0.15.7-rc5`
  * database backend mysql

### Server-side log
```bash
 ./bin/keygen -salt="T713/rYYgW7g4m3vG6zGRh7+FM1t0T8j13koXScOAj4="
2018/10/15 11:32:38 Failed to decode HMAC salt illegal base64 data at input byte 4
```

**Comment by or-else** [comment]
The string is expected to use base64 url-encoded. 
```
./bin/keygen -salt="T713_rYYgW7g4m3vG6zGRh7-FM1t0T8j13koXScOAj4="
API key v1 seq1 [ordinary]: AQAAAAABAAAoeOI7tA3HsYvdzDhYhZJy
Used HMAC salt: T713_rYYgW7g4m3vG6zGRh7-FM1t0T8j13koXScOAj4=
```

**Comment by or-else** [comment]
I've updated keygen. It can now take either base64 variant.

**Comment by or-else** [comment]
Yes, you are correct. It's an inconsistency. I have updated keygen to accept the standard base64 encoding as well as output the salt in std base64. It's in the `devel` branch and will be released shortly. 

Thanks!

**Comment by or-else** [comment]
The fix is released in v0.15.7
